allennlp/models/semantic_role_labeler.py

```diff
@@ -175,6 +183,14 @@ def tag(self, text_field: TextField, verb_indicator: IndexField) -> Dict[str, An
 
         return {"tags": tags, "class_probabilities": predictions.numpy()}
 
+    def get_metrics(self, reset: bool = False):
+        # TODO(Mark): This won't play well with the metrics in Trainer, see how Matt wants it.
```

Comment:
Done - I also added `Dict[str, float]` to the base class. I think it's impractical to pass this as anything other than a dict, really.